Documentation/kubernetes.md

```diff
@@ -119,6 +119,8 @@ To clean up all the artifacts created by the demo, run the following:
 kubectl delete -f wordpress.yaml
 kubectl delete -f mysql.yaml
 kubectl delete deployment rook-operator
```

Comment:
the order of deleting the rook operator doesn't actually matter. it can go away first and it won't impact anything else being deleted